ReactAndroid/src/main/jni/react/JSCExecutor.cpp

```diff
@@ -540,4 +549,16 @@ static JSValueRef nativePerformanceNow(
   return JSValueMakeNumber(ctx, (nano / (double)NANOSECONDS_IN_MILLISECOND));
 }
 
+static JSValueRef injectHMRUpdate(
+    JSContextRef ctx,
+    JSObjectRef function,
+    JSObjectRef thisObject,
+    size_t argumentCount,
+    const JSValueRef arguments[], JSValueRef *exception) {
+  JSStringRef execJSString = JSValueToStringCopy(ctx, arguments[0], NULL);
```

Comment:
These two lines leak memory. Use the String datastructure we have. (`Value(ctx, arguments[0]).toString()`)